core/src/test/java/io/parsingdata/metal/EqualityTest.java

```diff
@@ -196,24 +196,28 @@ public void parseGraph() {
 
     @Test
     public void stringRef() {
-        final Ref object = new NameRef("name");
-        assertFalse(object.equals(null));
+        final Ref<String> object = new NameRef("name");
+        assertNotEquals(null, object);
```

Comment:
IntelliJ was nagging about this line stating that `assertFalse(object.equals(null))` should be converted `assertNotEquals(null, object)`. Since the `AutoEqualityTest` also explicitly test for `equals` methods, I obliged IntelliJ here.